codegens/csharp-restsharp/test/unit/convert.test.js

```diff
@@ -179,7 +178,6 @@ describe('csharp restsharp function', function () {
                 options = {
                     indentCount: 1,
                     indentType: 'tab',
-                    requestTimeout: 2000,
```

Comment:
why have we removed this?

Reply:
@abhijitkane Previously it wasn't used in the code at all. After handling the timeout, some requests are getting bailed out. Will put it back by increasing the timeout to 5 seconds.